mac-setup-normal.sh

```diff
@@ -209,7 +209,7 @@ install_python2() {
     fi
 
     info "Installing python2 from khan/repo. This may take a few minutes."
-    brew install khan/repo/python@2
+    brew86 install khan/repo/python@2
```

Comment:
@dbraley do you have any idea why this would be a problem now and not with Monterey?

Reply:
To clarify, I haven't confirmed this is a problem. I just copied this over from the [khan-dotfiles setup guide](https://khanacademy.atlassian.net/wiki/spaces/INFRA/pages/1201864735/Setting+up+khan-dotfiles#Installing-khan-dotfiles-on-a-M1-mac). Let me know if this should be reverted.